blogposts/2020/software-engineer-career-paths.md

```diff
@@ -0,0 +1,85 @@
+
+Second, I was wrong about how to get that Senior title. At the time, I measured myself by the things I understood: how quickly I could implement features, the performance of my code, and how few bugs it had. I thought that my career progression would be about me improving those metrics. Write faster code faster, and with fewer bugs, and then I’d be recognized as a Senior Software Engineer.
+
+What I ultimately came to realize was that _the real measure of my career growth was in having an ever-increasing impact_. My focus at that time was very much on my personal performance, which was only one part of my impact.
+
+<div class="text-center">
+  <img src="https://sourcegraphstatic.com/blog/career/Impact.jpg" width="70%" alt="Advancing in my career illustration">
+</div>
+
+You can increase your impact, and thus grow in your career, in a number of different ways. You could expand the depth and breadth of your skills, to have a greater direct impact in more areas. You could increase the scope of your concerns to encompass the performance of your team, and eventually of your entire organization. You could work with, influence, and support a broader range of people. Each of these paths will come with a different set of titles, but chasing those titles is putting the cart before the horse.
+
+I’m not saying titles don’t matter. They do, depending on where you work. Having a title starting with Senior or Lead can affect how deeply people consider your input, or what projects and opportunities are available to you. If you are doing the same kind/quality/quantity of work as someone else, of course you will care if you don’t have the same title. But it is still a mistake to treat a string of titles as being identical to your career path.
+
+(One thing I love about working at Sourcegraph is that we don’t have those title prefixes. I “lost” my Senior title when I came here, and I wasn’t sure how I would feel about that. But right away I could see how it put early career engineers on a more equal footing with more experienced engineers, which raises the quality of conversations and designs. That’s definitely worth more to me than a title.)
+
+Every career is unique; there’s no one right way to do this. The trick is to find the path that most resonates with _you_. When you are the most engaged, you will be most effective in expanding your own impact. Here are a few anonymized examples of real software engineers and their career paths.
+
+<div class="text-center">
+  <img src="https://sourcegraphstatic.com/blog/career/Maria.jpg" width="70%" alt="Maria illustration">
+</div>
+
+## Maria, Engineer (Technical Breadth)
```

Comment:
This is up to you, but I think the `(Technical Breadth)` and other parentheticals, and `Gentle Reader` should all be lowercase (sentence-case rule in our style guide at https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#headings-and-subheadings).

If you think they are "titles" not "descriptions", then it's fine to keep them the way they are.